sdks/java/io/solace/src/test/java/org/apache/beam/sdk/io/solace/data/SolaceTest.java

```diff
@@ -0,0 +1,146 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.beam.sdk.io.solace.data;
+
+import java.nio.charset.StandardCharsets;
+import org.apache.beam.sdk.io.solace.data.Solace.Destination;
+import org.apache.beam.vendor.grpc.v1p60p1.com.google.protobuf.ByteString;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class SolaceTest {
+
+  Destination destination =
+      Solace.Destination.builder()
+          .setName("some destination")
+          .setType(Solace.DestinationType.TOPIC)
+          .build();
+  String messageId = "some message id";
+  long expiration = 123L;
+  int priority = 7;
+  Boolean redelivered = true;
+  Destination replyTo =
+      Solace.Destination.builder()
+          .setName("some reply-to destination")
+          .setType(Solace.DestinationType.TOPIC)
+          .build();
+  long receiveTimestamp = 123456789L;
+  Long senderTimestamp = 987654321L;
+  Long sequenceNumber = 27L;
+  long timeToLive = 34567890L;
+  String payloadString = "some payload";
+  ByteString payload = ByteString.copyFrom(payloadString, StandardCharsets.UTF_8);
+  String attachmentString = "some attachment";
+  ByteString attachment = ByteString.copyFrom(attachmentString, StandardCharsets.UTF_8);
+
+  @Test
+  public void testRecordEquality() {
```

Comment:
If you are using AutoValue, it automatically creates an equals method possibly removing the need for this test.

Reply:
True, thanks, I'll remove it.